### ISSUE TITLE
Builds are failling in COPR

### ISSUE THREAD

**Issue by rpitonak** [packit-service/hello-world#19]
Link to build: https://copr.fedorainfracloud.org/coprs/packit/packit-service-hello-world-14/build/1034045/

Error:
```
Building target platforms: x86_64
Building for target x86_64
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.T2GP0M
+ umask 022
+ cd /builddir/build/BUILD
+ cd /builddir/build/BUILD
+ rm -rf None-0.2.0.3.g08bfc38
+ /usr/bin/gzip -dc /builddir/build/SOURCES/hello-0.2.0.3.g08bfc38.tar.gz
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd None-0.2.0.3.g08bfc38
BUILDSTDERR: /var/tmp/rpm-tmp.T2GP0M: line 38: cd: None-0.2.0.3.g08bfc38: No such file or directory
RPM build errors:
BUILDSTDERR: error: Bad exit status from /var/tmp/rpm-tmp.T2GP0M (%prep)
BUILDSTDERR:     Bad exit status from /var/tmp/rpm-tmp.T2GP0M (%prep)
Child return code was: 1
```

**Comment by thrix** [comment]
#14 will fix this

**Comment by rpitonak** [comment]
solved by #21
